src/hub.html

```diff
@@ -162,21 +162,44 @@
                     class="interactable"
                     super-networked-interactable="counter: #counter; mass: 1;"
                     body="type: dynamic; shape: none; mass: 1;"
+                    auto-scale-cannon-physics-body
                     grabbable
-                    stretchable="useWorldPosition: true;"
+                    stretchable="useWorldPosition: true; usePhysics: never"
                     hoverable
                     duck
-                    sticky-object
+                    sticky-object="autoLockOnRelease: true;"
                 ></a-entity>
             </template>
 
+            <template id="interactable-model">
+                <a-entity
+                    gltf-model-plus="inflate: false;"
+                    class="interactable"
+                    super-networked-interactable="counter: #media-counter; mass: 1;"
+                    body="type: dynamic; shape: none; mass: 1;"
+                    grabbable
+                    stretchable="useWorldPosition: true; usePhysics: never"
+                    hoverable
+                    sticky-object="autoLockOnRelease: true; autoLockOnLoad: true;"
+                    auto-box-collider
+                    position-at-box-border="target:.delete-button"
+                    auto-scale-cannon-physics-body
+                >
+                    <a-entity class="delete-button" visible-while-frozen scale="0.08 0.08 0.08">
+                        <a-entity mixin="rounded-text-button" remove-object-button position="0 0 0"> </a-entity>
+                        <a-entity text=" value:Delete; width:2.5; align:center;" text-raycast-hack position="0 0 0.01"></a-entity>
+                    </a-entity>
+                </a-entity>
+            </template>
+
             <template id="interactable-image">
                 <a-entity
                     class="interactable"
                     super-networked-interactable="counter: #media-counter; mass: 1;"
                     body="type: dynamic; shape: none; mass: 1;"
+                    auto-scale-cannon-physics-body
                     grabbable
-                    stretchable="useWorldPosition: true;"
+                    stretchable="useWorldPosition: true; usePhysics: never"
```

Comment:
`usePhysics` allows `stretchable` to adjust the halfExtents/radius of cannon shapes. We don't want this because we handle this ourselves in `auto-scale-cannon-physics-body`. (Note that stretchable only knows to adjust the halfExtents/radius while stretching. Hence right now in prod local duck colliders don't grow when a remote client does the stretching.